README.md

```diff
@@ -180,6 +180,43 @@ NOTE that this functionality works on incremental models that use partitioning:
 }}
 ```
 
+#### Materialization
+
+Adapter supports all materializations provided by dbt-core.
+
+Additionally, adapter supports two `table_exists_strategy`  with `table` materialization `RENAME` and `DROP`.
+
+- `RENAME` - creates intermediate table, then renames the target to backup one and renames intermediate to target one.
+- `DROP` - drops and recreates a table. It overcomes table rename limitation in AWS Glue.
+
+
+By default `table` materialization uses `table_exists_strategy = 'RENAME'`, see an examples below how to change it.
+
+In model add:
+```jinja2
+{{
+  config(
+    materialized = 'table',
+    table_exists_strategy = 'DROP`
+  )
+}}
+```
+
+or in `dbt_project.yaml`:
+
+```yaml
+models:
+  path:
+    materialized: table
+    +table_exists_strategy: DROP
+```
+
+Using `table` materialization and `table_exists_strategy = 'RENAME'` with AWS Glue may result in below error:
```

Comment:
`table_exists_strategy` -> `on_table_exists` ?

This seems in line with https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/include/global_project/macros/materializations/models/incremental/on_schema_change.sql 

which is used for the `incremental` materialization in dbt core.

Reply:
Done